### PR TITLE
Save & Load method for aggregate 'set'

### PR DESCRIPTION
**Pull request by bkazic**
- Added `saveState` and `loadState` for `TStreamAggrSet`
- `saveJson()` returns names (IDs) of aggregates in a set

### CODE REVIEW

File: test/nodejs/streamaggr.js [hunk 1]
```diff
@@ -6270,3 +6270,204 @@ describe('Record switch aggregate', function () {
...
+        assert.throws(function () { set.save(fout) }, Error);
+        fout.close();
+
+        store.resetStreamAggregates();
+
+        var fin = qm.fs.openRead('aggrSet.bin');
+        assert.throws(function () { set.load(fin) }, Error);
+        fin.close();
+    });
+});
```
Comment (rupnikj) [inline test/nodejs/streamaggr.js]: Additional unit tests needed for save/load to cover edge cases (empty set, non-init aggregates).
Also, the test doesn't test unimplemented load properly (if save failed, load will always fail)

File: test/nodejs/streamaggr.js [hunk 2]
```diff
@@ -6463,11 +6463,163 @@ describe('Aggregate set', function () {
...
+
+        var set = new qm.StreamAggr(base, {
+            type: 'set',
+            aggregates: [tick.name, winbufvec.name, customAggr.name]
+        });
+
+        //store.push({ Time: '2015-06-10T14:13:32.0', Celsius: 1 });
+        //winbufvec.getFloatVector().print(); // prints 1
+        //store.push({ Time: '2015-06-10T14:33:30.0', Celsius: 2 });
+        //winbufvec.getFloatVector().print(); // prints 2
```
Comment (rupnikj) [inline test/nodejs/streamaggr.js]: remove commented code
